jetbrains-rider/build.gradle

```diff
@@ -1,6 +1,8 @@
 // Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import groovy.json.JsonSlurper
```

Comment:
Looks like this isnt needed anymore, same with change to gradle.properties

Reply:
Fixed here and in gradle.properties file.